metricbeat/docs/modules/aws.asciidoc

```diff
@@ -90,6 +90,73 @@ for a list of AWS services that publish metrics to CloudWatch.
 `period` for `rds` metricset is recommended to be `60s` or multiples of `60s` because Amazon RDS sends metrics and
 dimensions to Amazon CloudWatch every minute.
 
+[float]
+== AWS API requests count per metricset
+This session is to document what are the AWS API called made by each metricset
+in `aws` module. This will be useful for users to estimate costs for using `aws`
+module.
+
+Note: some AWS APIs need pagination like ListMetrics and GetMetricData. Count
+value is depends on the number of results.
+
+[float]
+=== `cloudwatch`
+|===
+| AWS API Name | AWS API Count | Frequency
+| IAM ListAccountAliases | 1 | Once on startup
+| STS GetCallerIdentity | 1 | Once on startup
+| EC2 DescribeRegions| 1 | Once on startup
+| Cloudwatch ListMetrics | Total Number of results / 500 | Per region per namespace per collection period
```

Comment:
nit: sometimes it's safer not to give values here. I'm pretty sure the next person who will change the page size will also forget about this docs references. Maybe instead use "/ page size" ?

Reply:
500 and 100 here are actually from AWS API documentation. I will document that with the api links. Thanks!